### DISCUSSION TITLE
deepinV23报错日志

### DISCUSSION THREAD

**Discussion by Linux-qitong** [linuxdeepin/developer-center#4250] [Troubleshooting | 故障排查]
https://bbs.deepin.org/phone/zh/post/255738
管窗频繁卡死，没办法删除rsyslog和coredump后，系统能够使用，查看日志有大量报错，看是否有价值

**Comment by BLumia** [comment]
v23 beta 应该是解决了的(https://github.com/linuxdeepin/developer-center/issues/4070 )，你目前是 alpha 2 的镜像吗？是的话，可以尝试参与下 beta 的内测。
